Project-Folder/SSS/app/src/main/java/coen390/nicholas/sss/quizTracking.java

```diff
@@ -2,11 +2,39 @@
 
 public class quizTracking
 {
+    //-------------------------------------Variables to set the lvl list---------------------------------------------
+    private String lvl;
+    private String lvlDescription;
+    private int totalQ = 0;
+
     //-------------------------------------tracking variables for quizzes---------------------------------------------
-    public int nmbCorrectQ1;
-    int questionCompletition[] = new int[26];
+    private static int nmbCorrect;
+    int questionCompletion[] = new int[26];
 
     //---------------------------------------------Quiz Questions-------------------------------------------------
-    String questions[] = {"A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T",
+    private static String questions[] = {"A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T",
             "U", "V", "W", "X", "Y", "Z"};
+
+    //----------------------------------------------------Constructor--------------------------------------------------------
+    public quizTracking(String level, String description)
+    {
+        lvl = "Level " + level; lvlDescription = description;
+
+       // if (level == "1"){totalQ = 26;}
+    }
+
+    //--------------------------------------------------------Getters-------------------------------------------------------4
+    //get the level
+    public String getLevel(){return lvl;}
+
+    //get the description
+    public String getDescription() { return lvlDescription;}
+
+    public String getScore() { return "Score: " + nmbCorrect + "/" + totalQ;}
+
+    public void setCorrect() { nmbCorrect += 1;}
+
+    public static int getNmbCorrect() { return nmbCorrect;}
```

Comment:
remove static

Reply:
thanks @prince-chrismc !

This corrected my issue